build/commands/lib/util.js

```diff
@@ -734,7 +732,7 @@ const util = {
     let cmd_options = config.defaultOptions
     cmd_options.cwd = config.braveCoreDir
     cmd_options = mergeWithDefault(cmd_options)
-    util.run('vpython', [path.join(config.braveCoreDir, 'build', 'commands', 'scripts', 'lint.py'),
+    util.run('vpython3', [path.join(config.braveCoreDir, 'build', 'commands', 'scripts', 'lint.py'),
```

Comment:
what is it for

Reply:
this will be landed separately https://github.com/brave/brave-browser/issues/25337